test/timeout_test.go

```diff
@@ -202,6 +203,20 @@ func TestTaskRunTimeout(t *testing.T) {
 	if err := WaitForTaskRunState(c, "run-giraffe", FailedWithReason("TaskRunTimeout", "run-giraffe"), "TaskRunTimeout"); err != nil {
 		t.Errorf("Error waiting for TaskRun %s to finish: %s", "run-giraffe", err)
 	}
+
+	tr, err := c.TaskRunClient.Get("run-giraffe", metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("Error retrieving TaskRun %s: %v", "run-giraffe", err)
+	}
+
+	for _, step := range tr.Status.Steps {
+		if step.Terminated == nil {
+			t.Errorf("TaskRun %s step %s does not have a terminated state but should", "run-giraffe", err)
```

Comment:
This should actually be the following:

```suggestion
			t.Errorf("TaskRun %s step %s does not have a terminated state but should", "run-giraffe", step.Name)
```